src/main/resources/META-INF/plugin.xml

```diff
@@ -20,40 +20,48 @@
     </extensions>
 
     <actions>
-        <!-- These actions are used in right-click context menu on branches. -->
-        <action id="GitMachete.CheckOutBranchAction"
-                class="com.virtuslab.gitmachete.frontend.actions.CheckOutBranchAction"
-                text="_Check Out Branch"
-                description="Check out this branch"/>
-
-        <action id="GitMachete.RebaseSelectedBranchOntoParentAction"
-                class="com.virtuslab.gitmachete.frontend.actions.RebaseSelectedBranchOntoParentAction"
-                text="_Rebase Branch Onto Parent"
-                description="Rebase selected branch onto parent"/>
-
-        <action id="GitMachete.SlideOutSelectedBranchAction"
-                class="com.virtuslab.gitmachete.frontend.actions.SlideOutSelectedBranchAction"
-                text="_Slide Out Branch"
-                description="Slide out branch"/>
-
         <group id="GitMachete.ContextMenu">
-            <reference id="GitMachete.CheckOutBranchAction"/>
-            <reference id="GitMachete.RebaseSelectedBranchOntoParentAction"/>
-            <reference id="GitMachete.SlideOutSelectedBranchAction"/>
+            <!-- These actions are used in right-click context menu on branches. -->
+            <action id="GitMachete.CheckOutBranchAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.CheckOutBranchAction"
+                    text="_Check Out Branch"
+                    description="Check out this branch"/>
+
+            <action id="GitMachete.RebaseSelectedBranchOntoParentAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.RebaseSelectedBranchOntoParentAction"
+                    text="_Rebase Branch Onto Parent"
+                    description="Rebase selected branch onto parent"/>
+
+            <action id="GitMachete.SlideOutSelectedBranchAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.SlideOutSelectedBranchAction"
+                    text="_Slide Out Branch"
+                    description="Slide out branch"/>
         </group>
 
-        <action id="GitMachete.RebaseCurrentBranchOntoParentAction"
-                class="com.virtuslab.gitmachete.frontend.actions.RebaseCurrentBranchOntoParentAction"
-                text="Rebase Current Branch Onto Parent"
-                description="Rebase current branch onto parent"
-                icon="AllIcons.Actions.Menu_cut"/>
+        <group id="GitMachete.Toolbar">
+            <action id="GitMachete.RefreshStatusAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.RefreshStatusAction"
+                    text="Refresh Status"
+                    description="Refresh status"
+                    icon="AllIcons.Actions.Refresh"/>
```

Comment:
If you add `use-shortcut-of="Refresh"` here we will be able to refresh our graph table with a key shortcut. (CTRL + F5)

Reply:
Done